gtsfm/scene_optimizer.py

```diff
@@ -176,10 +176,12 @@ def create_computation_graph(
         auxiliary_graph_list = []
 
         # ensure cycle consistency in triplets
-        i2Ri1_graph_dict, i2Ui1_graph_dict, v_corr_idxs_graph_dict = dask.delayed(
-            cycle_consistency.filter_to_cycle_consistent_edges, nout=3
+        i2Ri1_graph_dict, i2Ui1_graph_dict, v_corr_idxs_graph_dict, rcc_metrics_graph = dask.delayed(
+            cycle_consistency.filter_to_cycle_consistent_edges, nout=4
         )(i2Ri1_graph_dict, i2Ui1_graph_dict, v_corr_idxs_graph_dict, two_view_reports_dict)
 
+        metrics_graph_list.append(rcc_metrics_graph)
```

Comment:
You might want to also add these JSON files to the list of files the script to plot metrics from JSON files reads from.

Reply:
good point, should be added now